src/lib/isPostalCode.js

```diff
@@ -20,7 +21,7 @@ const patterns = {
   DK: fourDigit,
   DZ: fiveDigit,
   EE: fiveDigit,
-  ES: fiveDigit,
+  ES: ESValidator,
```

Comment:
Can just add the regex directly here instead of the variable.

Reply:
sure thing